tests/test_lint.nim

```diff
@@ -1,6 +1,26 @@
 import std/unittest
 import "."/lint/validators
 
+proc testIsFilesPattern =
+  suite "isFilesPattern":
```

Comment:
Could you also add test cases for file pattters with slashes? As in: sub-directories.